### DISCUSSION TITLE
Wrong transaction to Chia Blockchain Official Wallet Address

### DISCUSSION THREAD

**Discussion by pingolingoli** [Chia-Network/chia-blockchain#15717] [Support]
Hello,

Earlier, I made a wrong transaction to one of the wallet addresses that Chia Blockchain intermediates in transactions: xch1e7laam2ufj3du0gt75stnj6tka6r5dvm6tn2rzx3nnnallpp60ns44afhc. I am attaching the transaction number ([3,914,721](https://xchscan.com/blocks/3914721).. Please return a minimum of 0.33 XCH to my address. You may keep the remaining amount in the wallet, and during the Chia's halving event, as well as Bitcoin's halving event, please consider donating it to a cause of your choice.

![wrong](https://github.com/Chia-Network/chia-blockchain/assets/138997395/ff69f865-ae95-417f-81aa-eb42e2393a0e)

**Comment by BrandtH22** [comment]
Hey @pingolingoli , I am sorry for the delayed response. With our support moving from keybase to discord (https://discord.gg/chia) we have unfortunately not been as attentive of the github issues but we are working to improve our responses in all support channels.

CNI does not have any control or access to the xch addresses that are established for offer file completions and would not be able to return any of funds incidentally sent to one of those addresses.

I highly recommend updating your chia client to 1.8.2 which has a clawback feature that you can enable by default. When this is used you will have the time period of the clawback to retrieve your old funds but unfortunately this cannot work retroactively for claiming the funds accidentally sent to the address.

Please do let us know if you have any followup questions or would like to deep dive into the use of clawbacks to help avoid future transaction errors.
